insights/core/spec_factory.py

```diff
@@ -135,15 +135,15 @@ class TextFileProvider(FileProvider):
     Class used in datasources that returns the contents of a file a list of
     lines. Each line is filtered if filters are defined for the datasource.
     """
-    bad_lines = ["No such file or directory", "Command not found"]
-
-    @classmethod
-    def validate_lines(self, results):
-        if results and len(results) == 1:
-            first = results[0]
-            if any(l in first for l in self.bad_lines):
-                return False
-        return True
+    # bad_lines = ["No such file or directory", "Command not found"]
+    #
+    # @classmethod
+    # def validate_lines(self, results):
+    #     if results and len(results) == 1:
+    #         first = results[0]
+    #         if any(l in first for l in self.bad_lines):
+    #             return False
+    #     return True
```

Comment:
Make sure to remove the commented lines in your final commit.  Same with lines below.